copy-files-pipeline/acquia-pipelines.yaml

```diff
@@ -0,0 +1,39 @@
+version: 1.0.0
+
+variables:
+  global:
+    CLOUD_API_KEY:
+      secure: redacted
+    CLOUD_API_SECRET:
+      secure: redacted
+
+events:
+  build:
+    steps:
+      # If you have build instructions, put them here. They are not required.
+      - build_site:
+          script:
+            - touch docroot/index.html
+            - echo "Pipelines examples copy-files-pipeline. You successfully deployed your build to an ODE" > docroot/index.html
```

Comment:
We don't need to `touch` something that will be updated in the next command.

Reply:
I am keeping it the same to be consistent w/ tutorial-701 https://github.com/acquia/pipelines-examples/blob/master/tutorial-701/acquia-pipelines.yaml#L9.